### ISSUE TITLE
How do i generate my own Genesis block reward key 

### ISSUE THREAD

**Issue by KaptianCore** [tiagosh/AltcoinGenerator#26]
Hi I am changing the script like i am supposed to do but I am stuck on the public key and don't know how to make one or how I generate one please help?

**Comment by javier-lopez** [comment]
Hello, you can go to https://www.bitaddress.org/ to generate a new public/private pair, upon creation go to the Wallet Details section and copy the Public Key (130 characters [0-9A-F]) section. It's better described on this video: https://www.youtube.com/watch?v=jcu4BB7EAx0&feature=youtu.be&t=266

**Comment by Axe-Usa** [comment]
you got to do it?

**Comment by javier-lopez** [comment]
nope, however if you don't change it, you'll give your genesis block reward (50 coins) to the owner of this repository

**Comment by Axe-Usa** [comment]
hello javier, hola javier.
I have one question when you finish to create the crypto how i can cash it into my the wallet?
